tests/test_vqe.py

```diff
@@ -68,7 +68,7 @@ def seed():
     ((-0.8,), (qml.PauliZ(0),)),
     ((0.5, -1.6), (qml.PauliX(0), qml.PauliY(1))),
     ((0.5, -1.6), (qml.PauliX(1), qml.PauliY(1))),
-    ((0.5, -1.6), (qml.PauliX('a'), qml.PauliY('b'))),
+    ((0.5, -1.6), (qml.PauliX("a"), qml.PauliY("b"))),
```

Comment:
One day I'd love it if we ran `black` on all of our tests.

Reply:
Let's do it 💪